frostsnapp/native/src/api.rs

```diff
@@ -823,16 +951,31 @@ impl Wallet {
 
 pub struct SignedTx {
     pub inner: RustOpaque<RTransaction>,
+    pub psbt: Option<RustOpaque<bitcoin::psbt::PartiallySignedTransaction>>,
 }
 
 impl SignedTx {
     pub fn tx(&self) -> SyncReturn<RustOpaque<RTransaction>> {
         SyncReturn(self.inner.clone())
     }
+
+    pub fn save_psbt_to_file(&self, filename: String) -> Result<SyncReturn<()>> {
```

Comment:
It was easier to save the file using rust than return unknown length lists to flutter